src/templates/_loginForm.html

```diff
@@ -1,21 +1,21 @@
-<form class="form-signin" action="/login" method="post">
+<form class="form-signin" action="/login" method="post" data-testid="phovea_security_store_generated-form-signin">
```

Comment:
@dvvanessastoiber @dvflorianengertsberger Holger and I discussed the use of `data-testid` in this file and came to the conclusion that it is not required.
Example:
For selecting the username and password fields, one can use `.form-sign #login_username` (or even `#login_username` as it is a global id).
For selecting the submit button, one can use `.form-sign button[type="submit"]`.

Let me know if you have any questions, but this style of selecting nodes is preferred.